sql/src/test/java/io/crate/operation/udf/UserDefinedFunctionServiceTest.java

```diff
@@ -102,4 +107,18 @@ public void testReplaceIsFalse() throws Exception {
         expectedException.expectMessage("User defined Function 'same()' already exists.");
         putFunction(UserDefinedFunctionsMetaData.of(same1), same2, false);
     }
+
+    @Test
+    public void testInvalidFunction() throws Exception {
+        UserDefinedFunctionMetaData invalid = new UserDefinedFunctionMetaData(
+            "invalid", ImmutableList.of(), ImmutableList.of(), DataTypes.INTEGER,
+            "javascript", "function invalid(){ this is not valid javascript code }"
+        );
+        UserDefinedFunctionsMetaData metaData = UserDefinedFunctionsMetaData.of(invalid, same1);
```

Comment:
pls make same1, same2 and different function definition static and rename to uppercase. they can be static and usage more obvious (I was first wondering why 1 registered function is expected by that test ;)